### PR TITLE
Make Chakra Icon generator with @chakra-icons/cli

### PR DESCRIPTION
**Pull request by Akamig**
This works.... and also not works at the same time?

### CODE REVIEW

**Comment by rohjs** [comment]
Is this PR ready to review? @Akamig

File: packages/icons/scripts/transpile.js
Comment (rohjs) [inline packages/icons/scripts/transpile.js:1]: 👍

**Review by rohjs — APPROVED** [review]
LGTM 👍

**Comment by Akamig** [comment]
Closing for now, since we don't have more time on here.

**Pull request closed without merging**